api/src/shared/domain/errors.js

```diff
@@ -36,4 +41,10 @@ class CsvImportError extends DomainError {
   }
 }
 
-export { DomainError, ForbiddenAccess, EntityValidationError, CsvImportError };
+class NotFoundError extends DomainError {
+  constructor(message = 'Not found.') {
```

Comment:
```suggestion
  constructor(message = 'Erreur, ressource introuvable.') {
```
Reprendre l'existant